[FIX] 소셜 로그인 응답 형식 변경

jye326 opened the pull request:
## Issue Number
closed #964 

## As-Is
<!-- 문제 상황 정의 -->
- 소셜 로그인 성공의 응답 값과 일반 로그인 성공의 응답 값이 달라 채팅 등의 일부 기능에서 장애가 발생합니다.

## To-Be
<!-- 변경 사항 -->
- `/kakaoCallback` api의 로그인 성공 시 응답 값을 추가하였습니다. (`MemberLoginResponse`)
- AuthTokenResponse를 AuthTokenDto로 변경하여 명칭 일관성 향상
- 관련 서비스, 컨트롤러, 테스트 코드 수정 및 리팩토링
- 코드 가독성과 유지보수성 개선

## Check List
- [x] 테스트가 전부 통과되었나요?
- [x] 모든 commit이 push 되었나요?
- [x] merge할 branch를 확인했나요?
- [x] Assignee를 지정했나요?
- [x] Label을 지정했나요?
- [x] 닫을 이슈 번호를 지정했나요?


## (Optional) Additional Description
- 현재 로그인 관련 DTO의 네이밍 및 패키지 분류가 올바르지 않은 것 같습니다. 빠른 시일 내에 수정 PR을 올리겠습니다.

jye326 committed:
refactor: AuthTokenResponse를 AuthTokenDto로 클래스 이름 변경 및 관련 코드 수정

- AuthTokenResponse를 AuthTokenDto로 변경하여 명칭 일관성 향상
- 관련 서비스, 컨트롤러, 테스트 코드 수정 및 리팩토링
- 코드 가독성과 유지보수성 개선

ppparkta commented on backend/fittoring/src/main/java/fittoring/application/auth/presentation/dto/response/LoginResponse.java:
```diff
@@ -4,6 +4,6 @@
 public record LoginResponse(
         MemberLoginResponse memberLoginResponse,
-        AuthTokenResponse authToken
+        AuthTokenDto authTokenDto
```
p3: 이전에 비슷한 문제가 있어서 혹시나 해서 여쭤봅니다! `authToken`에서 `authTokenDto`로 key가 바뀌었다는 것이 클라이언트와 상의된 내용인가요? 명세서나 클라이언트 PR 확인해봤을 때에도 변경된 것이 없어서요...!

kimdevv replied:
그렇네요

jye326 replied:
LoginResponse와 AuthToken은 컨트롤러 단에서 사용되지 않아 기존의 컨벤션에 따라 Response라는 접미사를 제거해야 합니다. 같은 이유로 프론트에 따로 명세할 필요가 없습니다!

ppparkta reviewed (approved):
궁금한 점이 있어서 질문 하나 남기고 승인합니다. 
이미 상의된 내용이면 무시해주세요!

kimdevv reviewed (changes requested):
고생하셨습니다~ 커멘트 하나 확인해 주세요!

dye0p reviewed (approved):
크게 리뷰할게 없어서 바로 승인합니다! 혹시 dto 패키지 위치 관련 얘기는 더 안해봐도 괜찮을까요?

jye326 committed:
refactor: Auth 서비스와 DTO 구조 리팩토링

- `AuthService` 관련 DTO 패키지 재구성: service.dto와 presentation.dto.response로 리팩토링
- `LoginResponse` -> `LoginInfoDto`로 변경 및 관련 메서드, 테스트 수정
- `AuthService.login` 및 `kakaoLogin` 반환 타입 수정
- import 경로 수정 및 불필요한 코드 제거

jye326 committed:
Merge branch 'develop' of https://github.com/woowacourse-teams/2025-Fit-toring into fix/964-oauth-login
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

jye326 committed:
feat: AWS SQS 비활성화 설정 추가

- `application-local.yml`에 SQS 비활성화 설정(`cloud.aws.sqs.enabled: false`) 추가
- 로컬 환경에서 불필요한 SQS 연결 방지

jye326 committed:
feat: AWS SQS 비활성화 설정 추가

- `application-local.yml`에 SQS 비활성화 설정(`cloud.aws.sqs.enabled: false`) 추가
- 로컬 환경에서 불필요한 SQS 연결 방지

jye326 committed:
feat: 예약 정렬 기준 개선

- 멘토 ID로 예약 조회 시 `id` 기준 추가 정렬 지원
- `createdAt` 내림차순에 이어 `id` 내림차순으로 정렬
